README.rst

```diff
@@ -332,3 +332,11 @@ Root Cause: SignalFx's API endpoints (``api.signalfx.com``,
 the ``urllib3`` module in Python versions prior to 2.7.8 had a bug that causes
 the above issue. This was fixed in later versions of Python; we recommend using
 Python 2.7.9 or newer when using this library.
+
+Exceeding int64 integer sizes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+The protocol buffer used to transmit data through the ingest API restricts 
+integers and floats to (-9223372036854775808 - 9223372036854775807).  long 
```

Comment:
float64 does not have the same range that integers have. If you want to get very specific, it's approximately ±1.79769313486231570E+308  but ieee754 isn't even exact. You can refer people to the double protobuf documentation perhaps if they want exact values.